api.go

```diff
@@ -211,12 +225,42 @@ type (
 		Status TaskStatus `json:"status"`
 	}
 
+	createTranscodeTaskRequest struct {
+		Name    string  `json:"name,omitempty"`
+		AssetId string  `json:"assetId"`
+		Profile Profile `json:"profile,omitempty"`
+	}
+
+	CreateTranscodeTaskResp struct {
+		Asset Asset `json:"asset"`
+		Task  Task  `json:"task"`
+	}
+
+	Pinata struct {
+		JWT       string `json:"jwt,omitempty"`
+		APIKey    string `json:"apiKey,omitempty"`
+		APISecret string `json:"apiSecret,omitempty"`
+	}
+
+	IPFS struct {
+		Pinata *Pinata `json:"pinata,omitempty"`
+	}
+
+	createExportTaskRequest struct {
+		IPFS IPFS `json:"ipfs,omitempty"`
```

Comment:
This should be a pointer as well, since not every export is to IPFS and so this field may be `null`. (maybe `omitempty` wouldn't work either if it's not a pointer)